### PR TITLE
Angular

### PR DESCRIPTION
**Pull request by jcarolus**
* Angular module
* Added documentation for Angular in README

### CODE REVIEW

**Comment by BogetC** [comment]
Really need Angular Support. Hope this PR will be test soon.

File: README.md [hunk 1]
```diff
@@ -121,6 +121,42 @@ handleCastEvent(args): void {
...
+#### Angular
+
+Add the `NativeScriptModule` in your app module's `imports` 
+
+Typically in `app.module.ts`
+```ts
+import {NativescriptCastModule} from 'nativescript-cast/angular';
+
+@NgModule({
+    imports: [NativeScriptModule]
```
Comment (lochstar) [inline README.md]: Should this be `NativescriptCastModule` ?

File: README.md [hunk 2]
```diff
@@ -121,6 +121,42 @@ handleCastEvent(args): void {
 }
 ```
 
+#### Angular
+
+Add the `NativeScriptModule` in your app module's `imports` 
+
+Typically in `app.module.ts`
+```ts
+import {NativescriptCastModule} from 'nativescript-cast/angular';
```
Comment (lochstar) [inline README.md]: Can you add spaces around the brackets.

File: src/angular/nativescript-cast.module.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { NgModule } from "@angular/core";
+import {registerElement} from 'nativescript-angular/element-registry';
```
Comment (lochstar) [inline src/angular/nativescript-cast.module.ts]: Can you add spaces around brackets and change double quotes to single quotes.

File: README.md (hunk 1 above)
Comment (jcarolus) [reply README.md]: yes.. copy paste mistake. Perhaps I should just add a complete `angular-demo` folder with a working app, then that could be done later for Vue as well.

**Comment by jcarolus** [comment]
@lochstar I would like to finish the angular demo after the `expanded-controller` is on master; saves the time to update the Angular app twice...

**Comment by jcarolus** [comment]
@lochstar - removed WIP status, please check :)

**Pull request merged**